CakePHP/Tests/Commenting/DocBlockAlignmentUnitTest.1.inc

```diff
@@ -23,8 +23,9 @@ class Foo extends Bar
  *
  * @return void
  */
-    public function dumpThoughts()
+    public function dumpThoughts($thoughts)
     {
+        /** @var array $thoughts */
```

Comment:
Wouldn't the test be more thorough if there was a misaligned comment that the fixer could fix?